app/src/main/java/emu/skyline/EmulationActivity.kt

```diff
@@ -318,6 +320,17 @@ class EmulationActivity : AppCompatActivity(), SurfaceHolder.Callback, View.OnTo
                     }
                 }, 250)
             }
+
+            binding.ramStats.apply {
+                postDelayed(object : Runnable {
+                    override fun run() {
+                        updatePerformanceStatistics()
+                        ramUsage = File("/proc/self/statm").readLines()[0].split(' ')[1].toFloat() * 4096 / 1000000
```

Comment:
Tiny comment for future readers
```suggestion
                        // We read the `VmRSS` value from the kernel
                        ramUsage = File("/proc/self/statm").readLines()[0].split(' ')[1].toFloat() * 4096 / 1000000
```